libs/sr/formula/src/data/char/sheets/RuanMei.ts

```diff
@@ -41,6 +41,8 @@ const sheet = register(
 
   // Formulas
   ...dmg('basicDmg', baseTag, 'atk', dm.basic.dmg, 'basic'),
+  // TODO: should be Ice Break DMG
```

Comment:
It is

Reply:
oops, meant as a TODO to show the "ice variant" on the UI side. this TODO is in the wrong place